backend/src/test/scala/com/softwaremill/adopttapir/starter/StarterServiceITTest.scala

```diff
@@ -199,5 +197,5 @@ object ZipGenerator:
     yield service
 
 case class RunLogger(log: mutable.StringBuilder = new mutable.StringBuilder()):
-  def log(l: String): IO[Unit] = IO(log.append(System.lineSeparator()).append(l))
+  def log(l: String): IO[Unit] = IO(log.append(System.lineSeparator()).append(l).asInstanceOf[Unit])
```

Comment:
**suggestion:** This look suspicious. What's the type of `.append(l)`? I think it would be a bit nicer to call `.void` on `IO`:
```scala
IO(log.append(System.lineSeparator()).append(l)).void
```

Reply:
The result type of `append` is `StringBuilder` as it follows fluent API but I have counted on better suggestions 😸 🙇